contributing_guide.md

```diff
@@ -0,0 +1,37 @@
+## Contributing code commits
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+### Pull Request Process
```

Comment:
Here we should point people to specific procedures for our repo. Including what they should do if they want to contribute and how they should contribute. eg: When you file an issue, add the os version, app version etc. to the issue. For submitting code that there is an open issue for it, the cla is signed, tests pass etc. Issues tagged with help-wanted are good places to start, etc., etc.